### PR TITLE
update expected filenames for tls terminating key/secret

### PR DESCRIPTION
**Pull request by rsalmond**
https://istio.slack.com/archives/C3TEGNZ7W/p1695876172041579

https://github.com/istio/istio/blob/c9b1d29e5a1ac5a7a193c999627a56308eedc858/security/pkg/pki/ca/ca.go#L55-L58

I believe this is what it's supposed to say.

### CODE REVIEW

File: networking/v1alpha3/gateway.proto [hunk 1]
```diff
@@ -684,7 +684,7 @@ message ServerTLSSettings {
   // For gateways running on Kubernetes, the name of the secret that
```
Comment (howardjohn) [inline networking/v1alpha3/gateway.proto:684]: Both are valid, we should document both.

**Comment by linsun** [comment]
@rsalmond can you get make gen passing? Thx for your contribution - i didn't even know both are valid!

**Review by hzxuzhonghu — APPROVED** [review]

**Review by linsun — APPROVED** [review]
LG, thank you @rsalmond !

**Pull request merged**